### PR TITLE
[BIT] 引擎 accuracy 回测修复：__floordiv__ __matmul__ any argmax Tensor.argmax

### PR DESCRIPTION
**Pull request by Cutelemon6**
## paddle.Tensor.\_\_floordiv__

让 torch 对齐 paddle 输入为标量 tensor + 标量，输出为 shape=[1] 的情况，eg. 

```shell
paddle.Tensor.__floordiv__(Tensor([],"int64"), 8, )
```

解决办法是让 torch 输入的第二个参数包装为 torch.tensor([scalar])，这样输出 shape == [1]，而不是 []。为了保证不影响其它 case，第二个参数的 dtype 需要和输出 dtype 对齐，原始第二个输入为 int, float 时，需要确保与第一个输入 tensor 的 dtype 为同一位宽的 int, float，否则无需设置 dtype，比如第一个 tensor.dtype == 'bool'，第二个为标量 2 （int）的情况，这样 torch 和 paddle 的默认行为是一致的，让 torch 推断输出 tensor 的 dtype 即可。

配置均通过，移入 accuracy_8.txt

## paddle.Tensor.\_\_matmul__

pending_accuracy.txt / pending_accuracy_error.txt 的配置均确认为 accuracy error，移入 accuracy_eror_1.txt

## paddle.Tensor.any

paddle api 的 输入参数 axis == [] 或者 axis == () 时，此时 torch api 参数 dim 等价于 None，而不是对应的 []，或者 ()

```json
"torch_kwargs": {
    "dim": "None if locals().get('axis') is None or locals().get('axis') == [] or locals().get('axis') == () else axis"
},
```

配置均通过，移入 accuracy_8.txt

## paddle.Tensor.argmax & paddle.argmax

设置 `dtype` 为 int32 时，返回的 Tensor 的数据类型为 int32，其它情况返回的 Tensor 的数据类型为 int64。

```python
result = result.to(dtype=torch.int32 if dtype == torch.int32 else torch.int64)
```

dtype 为 paddle 独有的参数，在预处理中获取并在后处理中根据 dtype 进行处理。后处理需要在 rule.py 中添加，但是原来这两个 api 只需要在 mapping.json 中编写即可，为此将 ArgmaxRule 继承 BaseRule，使用 self.apply_generic() 获得 mapping.json 的原有配置。

```python
defaults_code, map_code = self.apply_generic()
```

编写 core 时 torch.Tensor 方法需替换为输入第一个 tensor 的变量名，该变量名取决于在 ana_api_info() 中获取的函数签名参数字典 paddle_args_dict。经查 paddle.Tensor 方法获取函数签名时，第一个 tensor 的关键字为 'x'，而不是 'self'。

```python
if self.torch_api.startswith("torch.Tensor."):
    # for paddle.Tensor method, first arg self correspond to x in paddle signature
    core = f"result = {self.torch_api.replace('torch.Tensor.', 'x.')}(**_kwargs)"
else:
    core = f"result = {self.torch_api}(**_kwargs)"
```

配置均通过，移入 accuracy_8.txt

### CODE REVIEW

**Review by mzj104 — APPROVED** [review]
LGTM

**Review by wanghuancoder — APPROVED** [review]
LGTM
